src/DetailsView/details-view-initializer.ts

```diff
@@ -314,6 +314,7 @@ if (isNaN(tabId) === false) {
                 getPropertyConfigById: getPropertyConfiguration,
                 collapsibleControl: CardsCollapsibleControl,
                 cardInteractionSupport: allCardInteractionsSupported,
+                navigator: new NavigatorUtils(window.navigator),
```

Comment:
The construction of NavigatorUtils happens in three places in this file (line 202, 233, 317). Maybe we can store it in a variable and reuse it.